### PR TITLE
remove signup buttons for inactive groups

### PR DESCRIPTION
**Pull request by berlintam**
closes https://github.com/rubycorns/rorganize.it/issues/434

It is also the end of an era.... bye bye `even_a_button?`. It's been a blast. But unfortunately, you make no sense anymore. 

What are you even doing? I no longer know. 

sad

### CODE REVIEW

File: app/helpers/groups_helper.rb [hunk 1]
```diff
@@ -5,10 +5,14 @@ def show_join_group_button?(person, group)
     !person.member_of?(group)
   end
 
-  def even_a_button?(group)
+  def group_open?(group)
     !group.full?
   end
 
+  def group_active?(group)
+    !group.inactive
```
Comment (til) [inline app/helpers/groups_helper.rb]: Couldn't this also use the variant with the question mark, `inactive?`? Active record usually adds both variants for boolean columns, and it's good practice to use the one with question mark where possible (a so-called predicate method).

Comment (til) [inline app/helpers/groups_helper.rb]: Actually, now that I think of it, those are attributes that would fit well with the model, don't you think?

Previously, it was about deciding if there should even be a button, which is out of the model's responsibility, but saying whether it is active or not, or full or open, could definitely be done within the model itself.

**Comment by til** [comment]
I'll miss you too, `even_a_button?` :sob:

File: app/helpers/groups_helper.rb (hunk 1 above)
Comment (berlintam) [inline app/helpers/groups_helper.rb]: yes! you're totally right. Sorry, I was stuck in helper-mode & trying to figure out what is even a button.

**Comment by sareg0** [comment]
(!)[http://new3.fjcdn.com/gifs/Ron+swanson+actually+happy_6cf6f3_4786756.gif]

**Pull request merged**